src/pymachine/utils.py

```diff
@@ -135,16 +145,11 @@ def to_dot(self):
         # lines.append('\tordering=out;')
         # sorting everything to make the process deterministic
         node_lines = []
-        for node, n_data in self.G.nodes(data=True):
+        for node in self.G.nodes():
```

Comment:
No, I didn't intend, my code was not up-to-date, and I had a conflict

2016-06-14 13:05 GMT+02:00 Gábor Recski <notifications@github.com>:

> In src/pymachine/utils.py
> <https://github.com/kornai/pymachine/pull/11#discussion_r66949724>:
>
> > @@ -135,16 +145,11 @@ def to_dot(self):
> >          # lines.append('\tordering=out;')
> >          # sorting everything to make the process deterministic
> >          node_lines = []
> > -        for node, n_data in self.G.nodes(data=True):
> > +        for node in self.G.nodes():
>
> @Eszti <https://github.com/Eszti> did you intend to remove this
> functionality? I added it recently, perhaps your code wasn't up-to-date?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kornai/pymachine/pull/11/files/bf75f7fed1a0349f4985eb1574693750f50d3c11#r66949724>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGc6ODaTwzeJ3idZYJvB-maqhUnj3Piiks5qLorggaJpZM4IYE3Z>
> .
>